shell.middle/src/main/java/org/metaborg/spoofax/shell/commands/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * Contains the logic for executing expressions using the core, or executing commands which are
+ * common accross frontends.
+ */
+package org.metaborg.spoofax.shell.commands;
```

Comment:
I thought the client module would also add commands, so that's why I thought that they should have the same package name.